superset-frontend/plugins/plugin-chart-echarts/src/constants.ts

```diff
@@ -53,8 +53,8 @@ export const LABEL_POSITION: [LabelPositionEnum, string][] = [
   [LabelPositionEnum.Right, 'Right'],
   [LabelPositionEnum.Bottom, 'Bottom'],
   [LabelPositionEnum.Inside, 'Inside'],
-  [LabelPositionEnum.InsideLeft, 'Inside left'],
-  [LabelPositionEnum.InsideRight, 'Inside right'],
+  [LabelPositionEnum.InsideBottomLeft, 'Inside left'],
+  [LabelPositionEnum.InsideBottomRight, 'Inside right'],
```

Comment:
In [lines 61 and 63](https://github.com/sid-indonesia/superset/pull/2/files#diff-bbf4598c738256137a39995b5cbbf3cf4e6612a339cb8a03f11be5493236f9b5R61-R63) there were actually other constant values that has these, I guess it would be better if the user of these constants (`LabelPositionEnum.InsideBottomLeft`, `LabelPositionEnum.InsideBottomLeft`) make use of those constants in lines 61 and 63.